ruma-events/tests/room_message.rs

```diff
@@ -24,11 +25,14 @@ use serde_json::{from_value as from_json_value, json, to_value as to_json_value}
 #[test]
 fn serialization() {
     let ev = MessageEvent {
-        content: MessageEventContent::Audio(AudioMessageEventContent {
-            body: "test".into(),
-            info: None,
-            url: Some("http://example.com/audio.mp3".into()),
-            file: None,
+        content: assign!(MessageEventContent::text_plain(""), {
```

Comment:
I don't like how we have to construct a different type to override it now. We should have an additional constructor on `MessageEventContent` (just `new`) that takes a `MessageType` parameter. We could also additionally implement `From<MessageType> for MessageEventContent` so the latter doesn't have to usually be imported (instead users can write `MessageType::Foo(/* ... */).into()`).

Reply:
Yeah, I agree, should I do both? Or just go for the `new`?

Reply:
Sure. `new` is important for discoverability, `From` is a nice convenience thing on top, not strictly needed but why not 🙂